src/coreclr/src/pal/inc/pal.h

```diff
@@ -1195,11 +1176,7 @@ CreateMutexExW(
 // CreateMutexExW: dwFlags
 #define CREATE_MUTEX_INITIAL_OWNER ((DWORD)0x1)
 
-#ifdef UNICODE
 #define CreateMutex CreateMutexW
-#else
-#define CreateMutex CreateMutexA
-#endif
 
 PALIMPORT
 HANDLE
```

Comment:
You can delete `#define OpenMutex  OpenMutexA` below. There is no OpenMutexA.

Reply:
Deleted. git-grep has found some other A APIs in pal.h which can be swept for the unused ones:

```sh
src/coreclr/src/pal/inc/pal.h:#define PAL_GetPALDirectory PAL_GetPALDirectoryA
src/coreclr/src/pal/inc/pal.h:#define MessageBox MessageBoxA
src/coreclr/src/pal/inc/pal.h:#define CreateFile CreateFileA
src/coreclr/src/pal/inc/pal.h:#define SearchPath  SearchPathA
src/coreclr/src/pal/inc/pal.h:#define CopyFile CopyFileA
src/coreclr/src/pal/inc/pal.h:#define DeleteFile DeleteFileA
src/coreclr/src/pal/inc/pal.h:#define MoveFileEx MoveFileExA
src/coreclr/src/pal/inc/pal.h:#define FindFirstFile FindFirstFileA
src/coreclr/src/pal/inc/pal.h:#define FindNextFile FindNextFileA
src/coreclr/src/pal/inc/pal.h:#define GetFileAttributes GetFileAttributesA
src/coreclr/src/pal/inc/pal.h:#define GetFullPathName GetFullPathNameA
src/coreclr/src/pal/inc/pal.h:#define GetTempFileName GetTempFileNameA
src/coreclr/src/pal/inc/pal.h:#define GetTempPath GetTempPathA
src/coreclr/src/pal/inc/pal.h:#define GetCurrentDirectory GetCurrentDirectoryA
src/coreclr/src/pal/inc/pal.h:#define CreateProcess CreateProcessA
src/coreclr/src/pal/inc/pal.h:#define CreateFileMapping CreateFileMappingA
src/coreclr/src/pal/inc/pal.h:#define OpenFileMapping OpenFileMappingA
src/coreclr/src/pal/inc/pal.h:#define LoadLibrary LoadLibraryA
src/coreclr/src/pal/inc/pal.h:#define LoadLibraryEx LoadLibraryExA
src/coreclr/src/pal/inc/pal.h:#define GetModuleFileName GetModuleFileNameA
src/coreclr/src/pal/inc/pal.h:#define OutputDebugString OutputDebugStringA
src/coreclr/src/pal/inc/pal.h:#define GetEnvironmentVariable GetEnvironmentVariableA
src/coreclr/src/pal/inc/pal.h:#define SetEnvironmentVariable SetEnvironmentVariableA
src/coreclr/src/pal/inc/pal.h:#define GetEnvironmentStrings GetEnvironmentStringsA
src/coreclr/src/pal/inc/pal.h:#define FreeEnvironmentStrings FreeEnvironmentStringsA
```

Reply:
@jkotas, I haven't tested yet but by reading the code, it seems like at least some of these A methods are not used in PAL. Is the removal of these from PAL worth pursuing (in a separate PR)?

Reply:
Yes, deleting unused pal methods is fine